service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/Descriptor.java

```diff
@@ -390,14 +390,14 @@ public String toString() {
     /**
      * Describes a topic call.
      */
-    public static final class TopicCall<Message> {
+    public static final class TopicCall<MESSAGE> {
```

Comment:
Changing the type parameter to all caps potentially breaks source compatibility, I'm not sure that that's a good enough reason to break the API.